pkg/cli/containers.go

```diff
@@ -39,11 +40,25 @@ func (a *Container) Run(cmd *cobra.Command, args []string) error {
 	out := table.NewWriter(tables.Container, a.Quiet, a.Output)
 
 	if len(args) == 1 {
-		app, err := c.ContainerReplicaGet(cmd.Context(), args[0])
+		app, err := c.AppGet(cmd.Context(), args[0])
 		if err != nil {
-			return err
+			// see if it's the name of a container instead
+			container, err := c.ContainerReplicaGet(cmd.Context(), args[0])
+			if err != nil {
+				return err
+			}
+			out.Write(container)
+		} else {
+			containers, err := c.ContainerReplicaList(cmd.Context(), &client.ContainerReplicaListOptions{App: app.Name})
+			if err != nil {
+				return err
+			}
+			for _, container := range containers {
+				if a.All || container.Status.Columns.State != "stopped" {
+					out.Write(container)
+				}
+			}
```

Comment:
nit: I would prefer this be DRY-ed up.

Reply:
I restructured this whole function as a switch statement and created a helper function to do the standard printing. 👍